framework/src/mesh/MooseMesh.C

```diff
@@ -1093,6 +1093,12 @@ MooseMesh::setSubdomainName(SubdomainID subdomain_id, SubdomainName name)
   getMesh().subdomain_name(subdomain_id) = name;
 }
 
+const std::string &
+MooseMesh::getSubdomainName(SubdomainID subdomain_id)
+{
+  return getMesh().subdomain_name(subdomain_id);
```

Comment:
what will happen if a subdomain does not have a name?

Reply:
I believe we just use a string of the number.